KEXPPower/Networking/NetworkManager.swift

```diff
@@ -195,42 +194,12 @@ public struct NetworkManager {
             }
         }
     }
-
-    public func getConfiguration(queue: DispatchQueue = .main, completion: @escaping ConfigurationCompletion) {
-        guard
-            let configurationURL = KEXPPower.configurationURL,
-            let data = try? Data(contentsOf: configurationURL)
-        else {
-                let error = NSError(
-                domain: "com.kexppower.error",
-                code: 0,
-                userInfo: [NSLocalizedDescriptionKey: "Failure retrieving config"]
-            )
-            
-            queue.async { completion(.failure(error)) }
-            return
-        }
-        
-        do {
-            let configuration = try JSONDecoder().decode(Configuration.self, from: data)
-            queue.async {  completion(.success(configuration)) }
-        } catch let error {
-            let error = NSError(
-                domain: "com.kexppower.error",
-                code: 0,
-                userInfo: [NSLocalizedDescriptionKey: error.localizedDescription]
-            )
-            
-            queue.async { completion(.failure(error)) }
-        }
-    }
     
-    public func getArchiveStreamURL(bitrate: String, timestamp: String?, queue: DispatchQueue = .main, completion: @escaping ArchiveCompletion) {
+    public func getArchiveStreamURL(timestamp: String?, queue: DispatchQueue = .main, completion: @escaping ArchiveCompletion) {
         var parameters = [URLQueryItem]()
-        parameters.append(URLQueryItem(name: "bitrate", value: bitrate))
         parameters.append(URLQueryItem(name: "timestamp", value: timestamp))
         
-        router.get(url: KEXPPower.streamingURL, parameters: parameters) { result in
+        router.get(url: KEXPPower.sharedInstance.streamingURL, parameters: parameters) { result in
```

Comment:
`bitrate` is no longer needed/used?

Reply:
bitrate is being ignored by the service, maybe this will change when the new 320 bitrate is introduced...but not needed now

Reply:
Not for archive at the moment

Reply:
Cool, thanks for the confirmation!